Makefile

```diff
@@ -50,32 +50,34 @@ undeploy:
 
 .PHONY: docker
 docker:
-	$(MAKE) -C build all
 	$(MAKE) -C manager docker-all
 	$(MAKE) -C secret-provider docker-all
 	$(MAKE) -C connectors docker-all
+	$(MAKE) -C test/dummy-mover docker-all
 
 # Build only the docker images needed for integration testing
 .PHONY: docker-minimal-it
 docker-minimal-it:
-	$(MAKE) -C build docker-dummy-mover
 	$(MAKE) -C manager docker-all
 	$(MAKE) -C secret-provider docker-all
+	$(MAKE) -C test/dummy-mover docker-all
 	$(MAKE) -C test/services docker
 
 .PHONY: docker-build
 docker-build:
-	$(MAKE) -C build docker-build-all
+	$(MAKE) -C test/dummy-mover docker-all
 	$(MAKE) -C manager docker-build
 	$(MAKE) -C secret-provider docker-build
 	$(MAKE) -C connectors docker-build
+	$(MAKE) -C test/dummy-mover docker-build-dummy-mover
 
 .PHONY: docker-push
 docker-push:
 	$(MAKE) -C build docker-push-all
```

Comment:
I don't understand the decision when "-all" is used. Make -C build docker-build-all is not used but docker-push-all is. How does this work?

Reply:
I unified the target names. Hope it's fine now.